app/debt/get-debts.js

```diff
@@ -23,7 +23,8 @@ const getDebts = async (includeAttached = false) => {
       'createdBy',
       'attachedDate',
       'paymentRequestId'
-    ]
+    ],
+    order: [['createdDate', 'DESC']]
```

Comment:
Can you add tests in `test/integration/narrow/routes/capture.test.js` to ensure this ordering is correctly displayed?

Reply:
added test to get-debts-test